src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -4,36 +4,35 @@
 
 public class StorageImpl<K, V> implements Storage<K, V> {
     private static final int MAX_ITEMS_NUMBER = 10;
-    private final Object[] keyArray;
-    private final Object[] valueArray;
+    private final Object[] keys;
+    private final Object[] values;
     private int counter;
 
     public StorageImpl() {
-        keyArray = new Object[MAX_ITEMS_NUMBER];
-        valueArray = new Object[MAX_ITEMS_NUMBER];
-        counter = 0;
+        keys = new Object[MAX_ITEMS_NUMBER];
+        values = new Object[MAX_ITEMS_NUMBER];
     }
 
     @Override
     public void put(K key, V value) {
         for (int i = 0; i < counter; i++) {
-            if (key != null && key.equals(keyArray[i])
-                    || key == keyArray[i]) {
-                valueArray[i] = value;
+            if (key != null && key.equals(keys[i])
+                    || key == keys[i]) {
+                values[i] = value;
                 return;
             }
         }
-        keyArray[counter] = key;
-        valueArray[counter] = value;
+        keys[counter] = key;
+        values[counter] = value;
         counter++;
     }
 
     @Override
     public V get(K key) {
         for (int i = 0; i < counter; i++) {
-            if (key == null && keyArray[i] == null
-                    || key != null && key.equals(keyArray[i])) {
-                return (V) valueArray[i];
+            if (key == null && keys[i] == null
```

Comment:
```suggestion
            if (key == keys[i]
```
in this case keys will be either identical, either both null